libtransmission/announcer.cc

```diff
@@ -1401,7 +1401,10 @@ void multiscrape(tr_announcer_impl* announcer, std::vector<tr_tier*> const& tier
     // batch as many info_hashes into a request as we can
     for (auto* tier : tiers)
     {
-        auto const* const scrape_info = tier->currentTracker()->scrape_info;
+        auto const* const current_tracker = tier->currentTracker();
+        TR_ASSERT(scrape_info != nullptr);
```

Comment:
Did you mean for this to be `current_tracker != nullptr`?

```suggestion
        auto const* const current_tracker = tier->currentTracker();
        TR_ASSERT(current_tracker != nullptr);
        if (current_tracker == nullptr)
        {
                    continue;
        }
```